convert-hf-to-gguf.py

```diff
@@ -240,6 +258,25 @@ def extra_f16_tensors(self, name: str, new_name: str, bid: int | None, n_dims: i
 
         return False
 
+    def parameter_count(self):
+        total_model_parameters = 0
+        for name, data_torch in self.get_tensors():
+            # Got A Tensor
+
+            # We don't need these
+            if name.endswith((".attention.masked_bias", ".attention.bias", ".rotary_emb.inv_freq")):
+                continue
+
+            # Calculate Tensor Volume
+            sum_weights_in_tensor = 1
+            for dim in data_torch.shape:
+                sum_weights_in_tensor *= dim
+
+            # Add Tensor Volume To Running Count
+            total_model_parameters += sum_weights_in_tensor
+
+        return total_model_parameters
```

Comment:
I think this won't always work (for example, it will overestimate the parameter count in Mamba models which have tied output weight with token embeddings, yet which still include both tensors in the original model files).

Hmm, this is hard because the file name needs to be known before writing, yet the true parameter count is only known after `modify_tensors` runs over all the tensors of the model. Not sure how to make this work also in `--no-lazy` mode.